### DISCUSSION TITLE
action-runner-controller on github enterprise

### DISCUSSION THREAD

**Discussion by a-cesari** [actions/actions-runner-controller#2912] [Questions]
Dear all,
I'm using Github enterprise 3.8.8 and the url is of this form: https://github.xxxx.io.
I actually have 2 questions.
1. Is the new replica set version working also for github enterprise 3.8.8 or should I use the legacy version? ( I tried with both and same error with both)
2. I'm getting the following error:
![image](https://github.com/actions/actions-runner-controller/assets/6312038/6f1084d4-9a5b-4746-8810-e832fc92b840)

I also tried to set the proxy variable (which however is already set inside each kube node) but always same result.
I can say that the url on which the POST request is done is correct (I can get the registration token by doing a curl and using the same PAT that I'm passing to ACR. We are deploying on a AWS EKS cluster which is inside a private VPC but has outbound internet access via proxy. As said the proxy inside the nodes is already configured and indeed we can pull containers from public docker for example. 
What could be?
Thanks a lot and please let me know if you need any additional detail.

**Comment by Link-** [comment]
Check the FAQs section
